packages/trackerless-network/tsconfig.browser.json

```diff
@@ -5,9 +5,5 @@
     "rootDirs": ["src", "test"],
     "noImplicitOverride": false
   },
-  "include": [
-    "src/**/*",
-    "generated/**/*",
-    "test/**/*"
-  ]
+  "include": ["src/**/*", "generated/**/*", "test/**/*"]
```

Comment:
Kind of like the earlier, ngl

Reply:
I believe one can simply write

```
["src", "generated", "test"]
```

A matter of preference. I don't mind so I'd go with the flow.